keylime-ansible-gcp/README.md

```diff
@@ -0,0 +1,73 @@
+# Ansible Keylime for Google Cloud
+Ansible role to deploy a Fedora 35 instance on the Google Cloud Platform with [Keylime](https://github.com/keylime/keylime) and the [rust agent](https://github.com/keylime/rust-keylime) against a Virtualized TPM.
+
+Contributions are welcome, should anyone wish to have this role provision other Linux distributions.
+
+For details on using Keylime, please consult the
+[project documentation](https://keylime-docs.readthedocs.io/en/latest/).
+
+For details on the Rust agent, please consult the [repository](https://github.com/keylime/rust-keylime).
+
+## Configuration
+1. Install Pip \
+`$ dnf install python3-pip python3-wheel`
+2.  Install dependecies for the Ansible - GCP module \
+`$ pip3 install requests google-auth ansible` \
+`$ pip3 install ansible --upgrade`
+3. [Create GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+4. Enable Compute Engine for this project. \
+To do so, select the naivgation menu (the three bars to the left of the GCP logo), hover over "APIs & Services", click "Dashboard", select "+ ENABLE APIS AND SERVICES", search for "Compute Engine API", select and enable. 
+4. [Create a GCP service account](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount) for ansible's use. 
+5. [Create and download keys linked to this service account](https://support.google.com/cloud/answer/6158849?hl=en&ref_topic=6262490#serviceaccounts&zippy=%2Cservice-accounts). Note: download the keys in JSON format.
+6. Create ssh key pair \
+` ssh-keygen -t rsa -f ~/.ssh/gcp_ansible_key` 
+7. Add the ssh public key to the Metadata section of Compute Engine in Google Cloud Platform. (Compute Engine>Settings>Metadata>SSH) 
+`#  cat ~/.ssh/gcp_ansible_key.pub`
+8. Add path to ssh private key to ansible config (`/etc/ansible/ansible.cfg`) \
+Example:
+```
+[defaults] 
+private_key_file = /home/.ssh/gcp_ansible_key
+```
+9.Run the script to set up the environment. 
+```
+./set_env_var.sh --help
```

Comment:
It looks funky, but we do need `. set_env_var.sh` here, otherwise the env vars don't get updated in the correct shell.

Reply:
Ah, nevermind! Your new version works great.